### ISSUE TITLE
Add alt texts for images

### ISSUE THREAD

**Issue by mahasneh** [jordanopensource/contributions-web#31]
**Describe the bug**
Images like `section-image` are missing an alternative text.

**Expected behavior**
Images should have an `alt` text for accessibility.
